html/controls/plt/plt-input.css

```diff
@@ -43,22 +43,24 @@ body:not(.AllowSelect) .PLT {
 .PLT.Team .Role:not(.OnTrack)>span.Num { display: none; }
 .PLT.Team .Role.OnTrack>span.Pos { display: none; }
 
-.PLT.Team .Warn1, .PLT.Team .Warn1 .Box0 { background-color: #ffffa0; }
+.PLT.Team .Warn1, .PLT.Team .Warn1 .Box0 { background-color: #ffff00; }
 .PLT.Team .Warn1:nth-child(4n+0), .PLT.Team .Warn1:nth-child(4n-1),
-.PLT.Team .Warn1:nth-child(4n+0) .Box0, .PLT.Team .Warn1:nth-child(4n-1) .Box0 { background-color: #ffff66; }
-.PLT.Team .Warn2, .PLT.Team .Warn2 .Box0 { background-color: #ffdc6d; }
+.PLT.Team .Warn1:nth-child(4n+0) .Box0, .PLT.Team .Warn1:nth-child(4n-1) .Box0 { background-color: #e8e800; }
+.PLT.Team .Warn2, .PLT.Team .Warn2 .Box0 { background-color: #ff9010; }
 .PLT.Team .Warn2:nth-child(4n+0), .PLT.Team .Warn2:nth-child(4n-1),
-.PLT.Team .Warn2:nth-child(4n+0) .Box0, .PLT.Team .Warn2:nth-child(4n-1) .Box0 { background-color: #ffd000; }
-.PLT.Team .Warn3, .PLT.Team .Warn3 .Box0 { background-color: #ff6666; }
+.PLT.Team .Warn2:nth-child(4n+0) .Box0, .PLT.Team .Warn2:nth-child(4n-1) .Box0 { background-color: #e87800; }
+.PLT.Team .Warn3, .PLT.Team .Warn3 .Box0 { background-color: #ff2020; }
 .PLT.Team .Warn3:nth-child(4n+0), .PLT.Team .Warn3:nth-child(4n-1),
-.PLT.Team .Warn3:nth-child(4n+0) .Box0, .PLT.Team .Warn3:nth-child(4n-1) .Box0 { background-color: #ff4040; }
+.PLT.Team .Warn3:nth-child(4n+0) .Box0, .PLT.Team .Warn3:nth-child(4n-1) .Box0 { background-color: #e80808; }
```

Comment:
The two Warn3 colours are a bit too close now.

Reply:
changed